frontend/src/newConsole/components/newLogos/Logos.tsx

```diff
@@ -20,7 +24,9 @@ export class LogosComponent extends React.PureComponent<Props, State> {
         contentLoading={this.props.loading}
         showSideBarEditComponent={false}
       >
-        <PreviewBox />
+        <PreviewBox>
+          <CourseOutlinePreview />
+        </PreviewBox>
```

Comment:
can you copy or move this line to https://github.com/open-craft/opencraft/blob/edcb1fa95dacf650a55a35e5b1382a477ebbc1fa/frontend/src/newConsole/components/newConsolePage/ConsolePage.tsx#L137 and deploy on staging, the intention is since https://stage.console.opencraft.com/newconsole/theming/footer is already in the new component can we just server the course outline pages for all new components

Reply:
@shimulch

Reply:
@farhaanbukhsh added & deployed to stage.